set.mm

```diff
@@ -62625,16 +62625,41 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
   $}
 
   ${
-    $d x A $.  $d x B $.  $d x F $.  $d x G $.
-    $( Equality theorem for function value.  (Contributed by NM,
-       29-Dec-1996.) $)
+    $d x y z w A $.  $d x y z w B $.  $d x y z w F $.  $d x y z w G $.
+    $( Equality theorem for function value.  (Contributed by NM, 29-Dec-1996.)
+       Avoid ~ ax-12 .  (Revised by Gino Giotto, 28-May-2024.) $)
     fveq1 $p |- ( F = G -> ( F ` A ) = ( G ` A ) ) $=
+      ( vx vy vz vw wceq cv wbr cio cfv cab cuni wcel wex abbidv df-uni 3eqtr4g
+      wa csn wel breq eqeq1d eleq2d anbi2d exbidv df-iota df-fv ) BCHZADIZBJZDK
+      ZAUKCJZDKZABLACLUJULDMZEIUAZHZEMZNZUNDMZUQHZEMZNZUMUOUJFGUBZGIZUSOZTZGPZF
+      MVEVFVCOZTZGPZFMUTVDUJVIVLFUJVHVKGUJVGVJVEUJUSVCVFUJURVBEUJUPVAUQUJULUNDA
+      UKBCUCQUDQUEUFUGQFGUSRFGVCRSULDEUHUNDEUHSDABUIDACUIS $.
+    $( $j usage 'fveq1' avoids 'ax-12'; $)
+
+    $( Equality theorem for function value.  (Contributed by NM, 29-Dec-1996.)
```

Comment:
The same for fveq2: No use of axioms ax-10 to ax-13 in the current version of set.mm.